### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by montjoie**

### CODE REVIEW

File: lava-slave/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ RUN apt-get update
 COPY scripts/stop.sh .
 COPY scripts/start.sh .
 
+# e2fsprogs is for libguestfs
```
Comment (khilman) [inline lava-slave/Dockerfile:9]: Looks like a stray comment.

**Review by khilman — CHANGES_REQUESTED** [review]
Reviewed-by: Kevin Hilman <khilman@baylibre.com>

Other than the minor nit on the stray comment, this looks good to me. 

Thanks for breaking up the big PR into smaller bits.

File: lava-slave/Dockerfile (hunk 1 above)
Comment (montjoie) [reply lava-slave/Dockerfile:9]: No, I add a comment missing like the one on lava-master/Dockerfile.
But e2fsprogs is already present on lava-slave/Dockerfile.
So it's not a stray comment.

**Review by khilman — APPROVED** [review]
OK, looks good to me now.

File: lava-slave/Dockerfile (hunk 1 above)
Comment (khilman) [reply lava-slave/Dockerfile:9]: OK, thanks for clarifying.

**Pull request merged**